### PR TITLE
[14.0][BPRT] edi_sale_oca: Backport from v16.0

### PR DESCRIPTION
**Pull request by thienvh332**

### CODE REVIEW

**Comment by QuocDuong1306** [comment]
Hi a @thienvh332, we should change `edi_disable_auto` to `disable_edi_auto` : [here](https://github.com/OCA/edi/blob/1de9d2b962ff075dc2cf88b9e725dcc9fd6948bc/edi_oca/models/edi_exchange_consumer_mixin.py#L50), right?

**Comment by QuocDuong1306** [comment]
We also should change the version to `14.0.1.0.0`, right?
Btw, I think you could add your address to the `CONTRIBUTOR.rst` to record your effort on this module (optional)

File: edi_sale_oca/components/process.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    def msg_order_existing_error(self):
+        return _("Sales order has already been imported before")
+
+    @property
+    def msg_order_created(self):
+        return _("Sales order %s created")
+
+    @property
+    def msg_generic_error(self):
+        return _("Something went wrong with the importing wizard.")
```
Comment (QuocDuong1306) [inline edi_sale_oca/components/process.py]:
```suggestion
```
I think we should remove this one, It was not used anywhere

**Comment by thienvh332** [comment]
Hello @QuocDuong1306 , I updated it. Could you take a look at this?

**Review by QuocDuong1306 — APPROVED** [review]
lgtm

Thank a Thien

**Pull request closed without merging**